scripts/globals/mobskills/bomb_toss.lua

```diff
@@ -9,6 +9,12 @@ require("scripts/globals/status")
 local mobskillObject = {}
 
 mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    local suicideCheck = math.random(0, 100)
+    if suicideCheck <= 15 then -- 15% chance to use bomb_toss_suicide if bomb_toss is picked (50%)
```

Comment:
Not an opposition to merging, but is there any data for this or is this just by feeling?

Reply:
Retail data / videos of % based on bomb_toss_suicide

Reply:
Are you okay with this being merged? I saw you had a concern about this check happening multiple times, wasn't sure if this is the implementation you wanted to use.